sections/builder.js

```diff
@@ -1035,13 +1035,39 @@ export default [
     type: "method",
     method: "orderBy",
     example: ".orderBy(column, [direction])",
-    description: "Adds an order by clause to the query.",
+    description: "Adds an order by clause to the query. column can be string or list of string and object.",
```

Comment:
"column" probably should be "Column"?
"can be string or list of string and object" -> do I understand correctly that what you mean is "can be string, list of strings or an object"? Or you mean "can be string, list of string or list of objects"?

Reply:
```
"column" probably should be "Column"?
```
`column` is a parameter for the method. Nevertheless, does it needs to be `Column`? 

```
"can be string or list of string and object" -> do I understand correctly that what you mean is "can be string, list of strings or an object"? Or you mean "can be string, list of string or list of objects"?
```
The latter would be right.
I will change this more clearly. Please understand me that I'm not English native :)

Reply:
1) Right, then probably you want to enclose it in backticks (``) for visual highlight.
2) Np, that's a very minor thing.

Thanks a lot for your contribution ;)

Reply:
This might be my first `real world` open source contribution.

and I thank you for guiding me very kindly.

👍